packages/contractkit/src/utils/tx-signing.test.ts

```diff
@@ -24,7 +24,7 @@ function generateAccountAddressFromPrivateKey(privateKey: string): string {
 
 testWithGanache('Transaction Utils', (web3: Web3) => {
   // describe('Transaction Utils', () => {
-  describe('Signer Testing', () => {
+  describe.skip('Signer Testing', () => {
```

Comment:
Is this supposed to be skipped?

Reply:
Yes. It's broken (test don't exit if we run this). It's being addressed in another issue https://github.com/celo-org/celo-monorepo/issues/811